test/plugin/test_in_watch_process.rb

```diff
@@ -35,4 +36,75 @@ def test_emit
     d.run(expect_emits: 1, timeout: 3)
     assert(d.events.size > 1)
   end
+
```

Comment:
Grouping windows test case may be better by:

```
sub_test_case "xxxx" do 
  def test_default
  ...
  def test_customized_keys
 ...
  def test_lookup_user
 ...
end
```

Reply:
Thank you for the advice.
I have fixed this in https://github.com/daipom/fluent-plugin-watch-process/pull/3/commits/0fcd97df35e7d90d9dd2953bb4ad056f7003deaf.